CMakeLists.txt

```diff
@@ -149,7 +149,17 @@ function(find_python_package PYPINAME NAME MIN_VERSION)
 endfunction()
 
 find_python_package(GitPython git 3.1)
-
+find_python_package(correctionlib correctionlib 2.1)
+execute_process(COMMAND correction config --cmake
+    OUTPUT_VARIABLE CORRECTION_LIB_ARGS
+    OUTPUT_STRIP_TRAILING_WHITESPACE)
+message(STATUS "Correctionline library path: ${CORRECTION_LIB_ARGS}")
+string(REPLACE -Dcorrectionlib_DIR= "" CORRECTIONLIBPATH ${CORRECTION_LIB_ARGS})
+message(STATUS "Correctionline library path: ${CORRECTIONLIBPATH}")
+find_package(correctionlib REQUIRED PATHS ${CORRECTIONLIBPATH})
+set(THREADS_PREFER_PTHREAD_FLAG ON)
+find_package(Threads)
+find_package(ZLIB)
```

Comment:
The `correctionlib` installation should be excluded here for now, since it is not ready yet.

Reply:
changed